content/en/apps/guides/offline/_index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Offline CHT Server"
+linkTitle: "Offline CHT"
+weight: 100
+description: >
+ Running an instance of CHT Core server with no internet connection
+---
+
+## Introduction
+
+The CHT is built as an offline first application. This applies to clients, either  browsers or Android applications, connecting to the CHT server.  The server itself assumes it has Internet connectivity to provide services such as DNS, software updates and general use connectivity.  This document explores what it looks like when the CHT server is offline without these services available.
+
+Running a CHT server offline requires no modifications to the CHT itself.  Instead, supporting services normally found online are replicated locally. 
+
+## Considerations
+
+An offline CHT server is most appropriate for a development environment.  There are serious implications to consider before deploying an offline server in a production environment which are most often ignored for development.  Offline solution should have a plan in place for:
+
+* Alerting - How will alerts be sent in the case of downtime or degraded service? 
```

Comment:
I don't think these need to be general considerations or considerations around CHT-Core. I'd like these to relate to all the extra pieces that aren't part of a normal deployment (medic-hosted & self-hosted).

Examples:
Alerting on what happens when the dhcp, or dns server fails. We would want to recommend redundancy, backups and alerting for all those extra pieces that go along with cht-core offline.

Reply:
I'm confused as to what changes you would like to see. It seems we'd want "Alerting" as a general topic that should be applied to both CHT and DNS, DHCP etc., right?  

Feel free to grab me on slack if that's easier!

Reply:
Now that we have elevated peripheral pieces of infrastructure to "critical", we need to ensure those particular pieces are reliable. I was thinking this document would be the appropriate place to only discuss those new pieces of infrastructure that this unique deployment requires, and how to ensure they meet our Product Supported Systems. 

We need to have details on how to monitor, alert, audit, and restore all the new pieces of infrastructure. I think the cht-core should remain in the other docs.